workers/data_refinery_workers/processors/smasher.py

```diff
@@ -338,7 +353,7 @@ def _smash(job_context: Dict) -> Dict:
         # Metadata to TSV
         if job_context["dataset"].aggregate_by == "EXPERIMENT":
             for title, keys in metadata['experiments'].items():
-                with open(smash_path + title + '_metadata.tsv', 'w') as output_file:
+                with open(smash_path + title + '/' + title + '_metadata.tsv', 'w') as output_file:
```

Comment:
@kurtwheeler good note, we were waiting to fill in the section around this as we expected it would change (https://github.com/AlexsLemonade/refinebio-docs/issues/4)